src/commands/content.ts

```diff
@@ -13,6 +13,8 @@ export const content: CommandDefinition = {
         description: makeLines([
             'In order to check which version of the A32NX you have installed, please open the MSFS Content Manager from the second page on the main menu and search for \'A32NX\'. '
```

Comment:
```suggestion
            'In order to check which version of the A32NX you have installed, please open the MSFS Content Manager from the second page on the main menu and search for \'flybywire\'. '
```

Reply:
I suggest using "flybywire" as otherwise a lot of liveries are also showing up as they use A32NX often.

Reply:
Commited :)